lambdas/data-migration1/src/index.ts

```diff
@@ -0,0 +1,134 @@
+import AWS from 'aws-sdk';
+import Knex from 'knex';
+
+import DynamoDbSearchQueue from '@cumulus/aws-client/DynamoDbSearchQueue';
+import { connection } from '@cumulus/db';
+import Logger from '@cumulus/logger';
+
+const {
+  Manager,
+} = require('@cumulus/api/models');
+const schemas = require('@cumulus/api/models/schemas');
+
+const logger = new Logger({ sender: '@cumulus/data-migration' });
+
+export interface HandlerEvent {
+  env?: NodeJS.ProcessEnv
+}
+
+export interface RDSCollectionRecord {
+  name: string
+  version: string
+  process: string
+  granuleIdValidationRegex: string
+  granuleIdExtractionRegex: string
+  files: string
+  // default will be set by schema validation
+  duplicateHandling: string
+  // default will be set by schema validation
+  reportToEms: boolean
+  sampleFileName?: string
+  url_path?: string
+  ignoreFilesConfigForDiscovery?: boolean
+  meta?: object
+  tags?: string
+  created_at: Date
+  updated_at: Date
+}
+
+const getRequiredEnvVar = (name: string, env: NodeJS.ProcessEnv): string => {
```

Comment:
This method already exists in db/config - is there a reason for not using that here?

Reply:
I don't totally love that a general utility for getting required env vars lives in the `db` package.  I hate dumping things in `common` and creating that dependency though

Reply:
`common` or "its own package" are the only options that seem to make sense. I don't have a problem with this living in `common`, especially as we've been pruning `common` down to utilities that make more sense.

Reply:
789f1e2f1c4a9884c365a6f3ff28a9a2064b1b0d

Reply:
👍  that seems fine, or at least resolves the concern.